drivers/wifi/esp_at/esp.c

```diff
@@ -721,6 +721,38 @@ MODEM_CMD_DEFINE(on_cmd_ready)
 	return 0;
 }
 
+#if defined(CONFIG_WIFI_ESP_AT_FETCH_VERSION)
+
+static int cmd_version_log(struct modem_cmd_handler_data *data,
+			   const char *type, const char *version)
+{
+	LOG_INF("%s: %s", type, version);
```

Comment:
Since the major purpose of that is development and debugging I would change that into `LOG_DBG`.

Reply:
Personally I think `LOG_INF` better fits here, as it serves informational purposes for the user. This is something that in lots of cases user won't have control over, especially with modules that come pre-programmed with ESP-AT firmware. The idea is to have those string appear by default for every user of ESP-AT. This will allow to easily define what is the firmware being used on the module that Zephyr tries to communicate with. If those are not needed, then those can be disabled with `CONFIG_WIFI_ESP_AT_FETCH_VERSION=n`.

Reply:
In that case, I would set that option as disabled by default, but I have no strong objections here.